### PR TITLE
chore(buildtool): Python3 port

### PR DESCRIPTION
**Pull request by ewiseblatt**
@jtk54

### CODE REVIEW

File: unittest/buildtool/bom_scm_test.py [hunk 1]
```diff
@@ -55,23 +55,23 @@ def test_pull_bom(self):
...
-      self.assertEquals(
+      self.assertEqual(
           self.repo_commit_map[repo_name][PATCH_BRANCH], at_commit)
 
       summary = scm.git.collect_repository_summary(git_dir)
       semver = SemanticVersion.make(BASE_VERSION_TAG)
       expect_version = semver.next(
           SemanticVersion.PATCH_INDEX).to_version()
-      self.assertEquals(expect_version, summary.version)
+      self.assertEqual(expect_version, summary.version)
```
Comment (jtk54) [inline unittest/buildtool/bom_scm_test.py]: Are these backwards compatible for python 2?

File: unittest/run_tests.sh [hunk 2]
```diff
@@ -10,9 +10,10 @@ cd $TEST_DIR/../..
 passed_tests=()
 failed_tests=()
 
-for test in `cd $TEST_DIR; find . -name \*_test.py -print`; do
+for test in `cd $TEST_DIR; find buildtool -name \*_test.py -print`; do
   echo "Running $test"
-  PYTHONPATH=$TEST_DIR/../pylib:$TEST_DIR/../dev python $TEST_DIR/$test
+#  PYTHONPATH=$TEST_DIR/../pylib:$TEST_DIR/../dev python $TEST_DIR/$test
```
Comment (jtk54) [inline unittest/run_tests.sh]: Is this here for documentation purposes?

File: unittest/buildtool/bom_scm_test.py (hunk 1 above)
Comment (ewiseblatt) [reply unittest/buildtool/bom_scm_test.py]: yes

File: unittest/run_tests.sh (hunk 2 above)
Comment (ewiseblatt) [reply unittest/run_tests.sh]: oops

**Pull request merged**